src/rct/manifest_commands.py

```diff
@@ -138,7 +138,7 @@ def __init__(self, name="cli", aliases=None, shortdesc=None, primary=False):
                 shortdesc=shortdesc, primary=primary)
 
     def _get_usage(self):
-        return _("%%prog %s [OPTIONS] MANIFEST_FILE") % self.name
+        return _("%%(prog)s {name} [OPTIONS] MANIFEST_FILE").format(name=self.name)
```

Comment:
I don't think that this change `%%prog` to `%%(prog)s` is correct, because output of rct cat-manifest --help is following in master:

```
[root@thinkpad-t580 subscription-manager]# PYTHONPATH=./src python -m subscription_manager.scripts.rct cat-manifest --help
Usage: rct.py cat-manifest [OPTIONS] MANIFEST_FILE

Print manifest information

Options:
  -h, --help    show this help message and exit
  --no-content  skip printing Content Sets
```

But your branch prints following help messages:

```
[root@thinkpad-t580 subscription-manager]# PYTHONPATH=./src python -m subscription_manager.scripts.rct cat-manifest --help
usage: %(prog)s cat-manifest [OPTIONS] MANIFEST_FILE

Print manifest information

optional arguments:
  -h, --help    show this help message and exit
  --no-content  skip printing Content Sets
```

Reply:
Simple reverting this change doesn't fix this issue. The `%%prog` hack is not compatible with `format()`.